packages/larva-patterns/modules/buy-now/buy-now.twig

```diff
@@ -0,0 +1,17 @@
+{% if c_lazy_image %}
```

Comment:
Can you add some extendable classes to the twig file? Like there are on the button `{{ button_classes }} {{ button_typography_class }} {{ button_background_color_class }} {{ button_color_class }} {{ button_width_class }}` That way when it needs to be extended with fonts or colors it can be dome on the prototype level in the theme.

Reply:
Updated